packages/core/drizzle/0113_quiet_wendigo.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "latitude"."spans" DROP CONSTRAINT "spans_span_id_unique";--> statement-breakpoint
+ALTER TABLE "latitude"."spans" ADD CONSTRAINT "spans_span_id_trace_id_unique" UNIQUE("span_id","trace_id");
```

Comment:
span ids are only required to be unique per trace, not globally, so changing the index